Gruntfile.js

```diff
@@ -0,0 +1,101 @@
+module.exports = function (grunt) {
+  'use strict';
+
+  require('matchdep').filterDev('grunt-*').forEach(grunt.loadNpmTasks);
+
+  function init () {
+
+    grunt.initConfig({
+      availabletasks: {
+        tasks: {
```

Comment:
We'll probably want to add the `localConfig` task back so you can customize config.js without danger of accidentally committing local changes.